backend/src/main/java/com/shipping/backend/config/AppConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package com.shipping.backend.config;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Getter
+@Setter
+@NoArgsConstructor
+public class AppConfiguration {
+
+    @Value("${ui-message.package-types}")
```

Comment:
Checkout @ConfigurationProperties, preferable in Spring Boot
[https://www.baeldung.com/configuration-properties-in-spring-boot](https://www.baeldung.com/configuration-properties-in-spring-boot)

Reply:
Can we work this during Saturday, I have some trouble because when you run test it doesn't use the same properties files, I know that there is @TestPorperty source but I wasn't able to make it work.

Reply:
Take a look at this:
[https://www.baeldung.com/spring-test-property-source](https://www.baeldung.com/spring-test-property-source)

Reply:
@ConfigurationProperties already applied, pending to use @TestPropertySource